test/e2e/storage/testsuites/provisioning.go

```diff
@@ -305,35 +305,78 @@ func (p *provisioningTestSuite) DefineTests(driver storageframework.TestDriver,
 					}
 					e2evolume.TestVolumeClientSlow(f, myTestConfig, nil, "", tests)
 				}
-				myTestCase.TestDynamicProvisioning()
+				t.TestDynamicProvisioning()
 			}(i)
 		}
 		wg.Wait()
 	})
 }
 
+// ProvisionStorageClass provisions a StorageClass from a spec, if the StorageClass already exists
+// then it returns it as a `computed` StorageClass, if it doesn't exist then it's created first
+// and then returned, if the spec is nil then we return the `default` StorageClass
+func ProvisionStorageClass(
```

Comment:
just a small nit: maybe use setup StorageClass (I choose set up instead of create mainly because in some cases, we don't create new Storageclass). typically we use provision for disks/volumes

Reply:
in persistent_volumes-local.go there's a private method called `setupStorageClass(config, &testMode)`, is your suggestion to use that method instead?

Reply:
I think that this suggestion is about renaming `ProvisionStorageClass` to `SetupStorageClass`, is that right?

Reply:
right